sdk/appconfiguration/Azure.Data.AppConfiguration/src/Models/CompositionType.cs

```diff
@@ -6,7 +6,7 @@
 
 namespace Azure.Data.AppConfiguration
 {
-    /// <summary> The composition type describes how the key-values within the snapshot are composed. The 'all' composition type includes all key-values. The &apos;group_by_key&apos; composition type ensures there are no two key-values containing the same key. </summary>
+    /// <summary> The composition type describes how the key-values within the snapshot are composed. The &apos;key&apos; composition type ensures there are no two key-values containing the same key. The &apos;key_label&apos; composition type ensures there are no two key-values containing the same key and label. </summary>
```

Comment:
I've copied this from the generated code. Do you think we should update the generator to fix this?

Reply:
We've talked about that before, and I don't remember where we landed.   

@m-nash:  Thoughts?   Are we handling this better in tsp?

Reply:
We did some crude "escaping" but we have another issue here to wholistically tackle this since the `///` docs don't exactly follow common xml escaping rules we kind of have to write our own.  https://github.com/Azure/autorest.csharp/issues/3054